hack/runtime-migrator/README-rr.md

```diff
@@ -1 +1,178 @@
-# Runtime Restorer
\ No newline at end of file
+# Runtime Restore
+
+The `runtime-restore` application has the following tasks:
+1. Connect to a Gardener project, and a KCP cluster.
+2. Retrieve all existing shoot specifications.
+3. For each runtime on the input list:
+   1. Gets the Shoot, Cluster Role Bindings and OpenIDConnect resources from the backup.
+   2.  To prevent KIM from modifying the shoot and the runtime resources, update the Runtime CR to label the runtime as controlled by Provisioner.
+   3. Patch the shoot with the Shoot spec read from the backup.
+   4. Apply the ClusterRoleBindings from the backup if the objects don't exist on the runtime.
+   5. Apply OpenIDConnect from the backup if the objects don't exist on the runtime.
+
+## Build
+
+To build the `runtime-restore` application, run:
+
+```bash
+go build -o ./bin/runtime-restore ./cmd/restore
+``` 
+
+## Usage
+
+### Dry Run
+```bash
+./bin/runtime-restore \
+  -gardener-kubeconfig-path=/Users/myuser/gardener-kubeconfig.yml \
+  -gardener-project-name=kyma-dev  \
+  -kcp-kubeconfig-path=/Users/myuser/kcp-kubeconfig.yml \
+  -output-path=/tmp/ \
+  -dry-run=true \
+  -input-file-path=input/runtimeIds.txt \
+  -input-type=txt \
+  -backup-path=/Users/myuser/backup/results/backup-2025-01-10T09:27:49+01:00
+```
+
+This execution example does the following:
+1. Take the input from the `input/runtimeIds.txt` file (each row contains single `RuntimeID`).
+1. Proceed only with fetching the `Shoot`, `Cluster Role Binding`, and `OpenIDConnect` resources from the backup directory.
+1. Save the output files in the `/tmp/<generated name>` directory. The output directory contains the following:
+   - `restore-results.json` - the output file with the restore results
+
+### Backup and Switch Runtime to Be Controlled by KIM
+
```

Comment:
consider switching the order of code block and explainer text like you did in https://github.com/kyma-project/infrastructure-manager/pull/596/files#r1916205022

Reply:
done